source/Octopus.Server.Client/Model/ProjectResource.cs

```diff
@@ -125,6 +125,10 @@ public ProjectResource(string id, string name, string slug) : this()
         [Writeable]
         [Obsolete("Use " + nameof(DeploymentSettingsResource) + " instead on the `deploymentsettings` API.")]
         public string DeploymentChangesTemplate { get; set; }
+
+        [Writeable]
+        [Obsolete("Use " + nameof(DeploymentSettingsResource) + " instead on the `deploymentsettings` API.")]
```

Comment:
Should this Obsolete attribute be removed?

Reply:
Nope, I don't think so. I think that Obsolete is there to guide the users to use the newer deploymentSettings API (where, this new flag belongs) rather than directly via the project resource. We'd still need to have this on the ProjectResource itself.